src/storage/src/hummock/state_store.rs

```diff
@@ -296,7 +297,12 @@ impl StateStore for HummockStorage {
 
     define_state_store_associated_type!();
 
-    fn get<'a>(&'a self, key: &'a [u8], epoch: u64) -> Self::GetFuture<'_> {
+    fn get<'a>(
+        &'a self,
+        key: &'a [u8],
+        epoch: u64,
+        _vnode: Option<VirtualNode>,
+    ) -> Self::GetFuture<'_> {
```

Comment:
Why add `_vnode`? Is `get_with_vnode_set` not enough?

Reply:
This is the interface of `StateStore`. Streaming passes vnode here to storage. This `get` will call some inner function to do point-get.

Reply:
Btw, as is discussed before, we may not use `get_with_vnode_set` to do point get, due to its inefficiency. We might have to find out another way (e.g. record vnode -> parallel unit) to do vnode-pruned point-get.